grpc/python_client/fly_mission.py

```diff
@@ -42,13 +42,16 @@ def run():
                                         yaw=-60,
                                         camera_action=0))
 
-    stub.AddMission(dc.Mission(mission_items=mission_items))
+    stub.SendMission(dc.Mission(mission_items=mission_items))
```

Comment:
This will be a blocking call, as server is  waiting for call back from dronecore to write the result.

Reply:
we can have similar async call as below.

Reply:
Hm, ok so what is your suggestion? I'm not sure I understand your concern.